.github/ISSUE_TEMPLATE/create-new-issues-on-trac.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Create new issues on Trac
+about: Create new issues on Trac
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# NEW ISSUES SHOULD BE CREATED ON TRAC
+
+From this point on any new issues should be opened on the core trac instead of at GitHub to save them needing to be transitioned over there manually at a later point.
+
+Please prefix tickets with Twenty Twenty: so that they are easy to find over there.
```

Comment:
Probably better to make this a bit more clear like this:

```suggestion
Please prefix tickets with `Twenty Twenty:` so that they are easy to find over there.
```